pkg/kubelet/kubelet.go

```diff
@@ -962,6 +965,7 @@ type Kubelet struct {
 	nodeName        types.NodeName
 	runtimeCache    kubecontainer.RuntimeCache
 	kubeClient      clientset.Interface
+	restConfig      *rest.Config
```

Comment:
Can we remove this field? It's not unused anymore. The plugin uses kubeclient instead of building a dynClient (I'm not sure because I cannot see it again).

Reply:
Yes.

Reply:
Removed and pushed (https://github.com/kubernetes/kubernetes/compare/bfe3552d5dc1675e33d535218437a5511d718f20..ef96e143f8cbf8b23c785c9962bd4d6e506a14c7).